test/files/run/t6329_repl_bug.check

```diff
@@ -7,7 +7,7 @@ import scala.reflect.runtime._
 
 scala> implicitly[scala.reflect.ClassManifest[List[_]]]
 warning: 1 deprecation (since 2.10.0); for details, enable `:setting -deprecation` or `:replay -deprecation`
-val res0: scala.reflect.ClassTag[List[_]] = scala.collection.immutable.List[<?>]
+val res0: scala.reflect.ClassTag[List[_]] = scala.collection.immutable.List[java.lang.Class]
```

Comment:
This is a bit weird but it could be easily fixed if we change `ExistentialType.rewrap`